.github/workflows/release.yaml

```diff
@@ -207,8 +207,6 @@ jobs:
       with:
         fetch-depth: 0
 
-    # Important: update actions/download-artifact to v4 only when generator_generic_slsa3.yml is also compatible.
-    # See https://github.com/slsa-framework/slsa-github-generator/issues/3068
     - name: Download provenance
       uses: actions/download-artifact@8caf195ad4b1dee92908e23f56eeb0696f1dd42d # v4.1.5
```

Comment:
Do we want to move this to v4.3.3 also?

Reply:
The latest version is [v4.1.7](https://github.com/actions/download-artifact/releases/tag/v4.1.7), but I guess we can leave that for the next release?

Reply:
You mean Dependabot will pick this up next Monday anyway?

Reply:
Yes